test.shape

```diff
@@ -0,0 +1,11 @@
+1	1.8567
+2	0.5739
+3	0.8769
+4	0.7598
+5	0.9977
+&&
```

Comment:
The delimiting character for both input sequences for the gapc compiled binary is `+`. Vienna RNA uses `&` to separate two sequences for programs like `RNAcofold`. Do you @kmaibach or @Jasmin-Walter know if they use any delimiter in "shape" files? We should avoid introducing another variant if there are already solutions out there.

Reply:
After searching through their github repositories I cannot find any example of how the probing data file has to look like for two RNA sequences. Sadly, the webservice of RNAcofold does not provide the option to use the integration of probing data - so I could not find any examples here as well.

I tried working through the different code files but that would take much more time to figure out how everything works.

I guess that they do not have a special delimiter. From what I saw I figure they have one file with the probing data for both sequences without any delimiter at all. But that's just an educated guess. We'd probably need to test it with a local version of RNAcofold. Do we have it installed somewhere? 

For now I'd probably either go back to the way it was or alternatively we forego the delimiter altogether and put the probing information of the two sequences one after the other in one file.

Reply:
The problem with the previous solution was that I use a seperator (position) to split the probingData into two vectors (for leftBase and offsetBase). If a value of the probing data of 0.0 is used as seperator, errors can occur if other values besides the separator are also 0.0. 
I don't know how exactly the probing data is generated, but would it be possible to have the position in the first column of the file start again at 1 for the second sequence? This way we could check where the second sequence starts without using a delimiter.
For example for the test_rnahybrid.shape it would look like this:

1	1.0000
2	0.5768
1	0.8769
2	0.7598

Reply:
Ah, it's because in line 308 you test if the value is 0, right?
I deliberately did not include such a check because, as you say, it will throw errors if some other position has a value of 0 as well.

The solution I used isn't optimal either, because if you have a probing file like this:
1 1.000
2 0.5678
3 0.000
4 0.5643
5 0.7848

both inputs `AG UG` and `A UGG` are valid - even though the second one shouldn't be.
Therefore your idea to use a delimiter like `&&` is definitely better to account for such mistakes the user can make. 

@sjanssen2 what's your take on that? Keep the old solution (without any checks) for now and open an issue to take care of it later? Or should we discuss the options we have now?